README.md

```diff
@@ -566,6 +566,21 @@ systemctl start myapp@instance-1
 systemctl start myapp@instance-2
 ```
 
+If you use stateful failover, you need to start application stateboard.
```

Comment:
I think that `stateboard` should be wrapped in braces `"`. I think you should be quite experienced user to understand/know what stateboard is.

Also may be we should paraphrase `If you use stateful failover` to `If you plan/want to use stateful failover`.